platform/viewer/cypress/support/commands.js

```diff
@@ -83,7 +87,11 @@ Cypress.Commands.add('isPageLoaded', (url = '/viewer/') => {
 
 Cypress.Commands.add('openStudyList', () => {
   cy.initRouteAliases();
-  cy.visit('/');
+  cy.visit('/', {
+    onBeforeLoad: win => {
+      win.sessionStorage.clear();
+    },
+  });
```

Comment:
I'm guessing this is slightly faster than having Cypress click the `clear` button? (reduces requests by 1)

Reply:
I would say yes.. and much more safer...